operations/admin.go

```diff
@@ -171,3 +173,42 @@ func setServiceFlagValues(args []string, target bool, flags *model.APIServiceFla
 
 	return catcher.Resolve()
 }
+
+func viewSettings() cli.Command {
+	return cli.Command{
+		Name:   "get-settings",
+		Usage:  "view the evergreen configuration settings",
+		Action: doViewSettings(),
+	}
+}
+
+func doViewSettings() cli.ActionFunc {
```

Comment:
test for this?

Reply:
There isn't much to test since it does some boilerplate things and calls a rest route which already has tests. I could write a test to make sure that this doesn't error but I don't think that would be very useful